src/ResourceManager/DataFactoryV2/Commands.DataFactoryV2/Changelog.md

```diff
@@ -19,6 +19,16 @@
 -->
 
 ## Current Release
+Azure Data Factory new capabilities now fall under General Availability SLA. ADF has made cloud data integration easier than ever before. Build, schedule and manage data integration at scale wherever your data lives, in cloud or on-premises, with enterprise-grade security.?Accelerate your data integration projects with over 70 data source connectors available. Transform raw data into finished, shaped data that is ready for consumption by BI tools or custom applications. Easily lift your SQL Server Integration Services (SSIS) packages to Azure and let ADF manage your resources for you so you can increase productivity and lower TCO.?Meet your security and compliance needs while taking advantage of extensive capabilities and paying only for what you use. The ADF GA SDK changes includes the following:
```

Comment:
Please only have details about the changes to PowerShell here.  People looking for SDK details can go to the nuget page for the SDK package.